.github/workflows/golangci-lint.yml

```
@@ -0,0 +1,44 @@
+          # Optional: golangci-lint command line arguments.
+          args: --timeout=15m -v
+
+          # Optional: show only new issues if it's a pull request. The default value is `false`.
+          only-new-issues: false
+
+          # Optional: if set to true then the all caching functionality will be complete disabled,
+          #           takes precedence over all other caching options.
+          # skip-cache: true
+
+          # Optional: if set to true then the action don't cache or restore ~/go/pkg.
+          skip-pkg-cache: true
```

actionlint (1.7.4)

input "skip-pkg-cache" is not defined in action "golangci/golangci-lint-action@v6". available inputs are "args", "cache-invalidation-interval", "github-token", "install-mode", "only-new-issues", "problem-matchers", "skip-cache", "skip-save-cache", "version", "working-directory"

(action)